core/templates/pages/story-editor-page/services/story-editor-state.service.spec.ts

```diff
@@ -178,12 +178,9 @@ describe('Story editor state service', function() {
   it(
     'should fire an init event and set the topic name after loading the ' +
     'first story', function() {
```

Comment:
Add a new beforeEach and afterEach
```
const storyInitializedSpy = jasmine.createSpy('storyInitialized');
const storyReinitializedSpy = jasmine.createSpy('storyReinitialized');
  beforeEach(() => {
    testSubscriptions = new Subscription();
    testSubscriptions.add(storyEditorStateService.onStoryInitialized
      .subscribe(storyInitializedSpy);
    testSubscriptions.add(storyEditorStateService.onStoryInitialized
      .subscribe(storyReinitializedSpy);
  });

  afterEach(() => {
    testSubscriptions.unsubscribe();
  });
```
In the it blocks, where ever the .emit / .next is called, add : `expect(storyInitializedSpy).toHaveBeenCalled();`

Reply:
Done